sermant-backend/src/main/java/io/sermant/backend/entity/mount/MountConfig.java

```diff
@@ -14,37 +14,42 @@
  * limitations under the License.
  */
 
-package io.sermant.backend.entity;
+package io.sermant.backend.entity.mount;
+
+import lombok.Getter;
+import lombok.Setter;
 
 /**
- * Event Level
+ * Mount information
  *
- * @author xuezechao
- * @since 2023-03-02
+ * @author zhp
+ * @since 2024-08-22
  */
-public enum EventLevel {
+@Getter
+@Setter
+public class MountConfig {
     /**
```

Comment:
you can create a new file instead of modify original file. And does MountConfig  is also suitable for future agent function？

Reply:
done